sql/database.sql

```diff
@@ -9638,3 +9638,16 @@ CREATE TABLE `codes_history` (
   `update_by` varchar(255),
    PRIMARY KEY (`log_id`)
 ) ENGINE=InnoDB;
+
+-- Table to store multiple db connection details
+DROP TABLE IF EXISTS `zf_multiple_db`;
+CREATE TABLE `zf_multiple_db` (
+    `id` int(11) NOT NULL,
+    `namespace` varchar(255) NOT NULL,
+    `username` varchar(255) NOT NULL,
+    `password` text NOT NULL,
```

Comment:
make text bare or will break on modern mysql(note text can not be null and is always non null):
``````
`password` text,
``````